src/helpers/generate-path-matrix.ts

```diff
@@ -11,34 +11,47 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import * as core from '@actions/core';
 import { chunk, uniq } from 'lodash';
 import { context } from '@actions/github';
 import { getChangedFilepaths } from '../utils/get-changed-filepaths';
 import micromatch from 'micromatch';
 
 export class GeneratePathMatrix {
-  paths = '';
+  paths?: string;
+  globs?: string;
   override_filter_paths?: string;
   override_filter_globs?: string;
   paths_no_filter?: string;
   batches?: string;
 }
 
 export const generatePathMatrix = async ({
-  paths,
+  paths = '',
+  globs = '',
+  /** paths that override the changed files filter, causing the action to return all paths */
   override_filter_paths,
   override_filter_globs,
+  /** paths that will be returned regardless of their adherence to the filter */
   paths_no_filter,
+  /** number of evenly-sized batches to separate matching paths into (returns comma-separated result) */
   batches
 }: GeneratePathMatrix) => {
+  if (!paths && !globs) {
+    const err = 'Must supply one of paths, globs';
+    core.error(err);
+    throw new Error(err);
```

Comment:
Nit pick, but probably can just do this so the error doesn't log twice, and to be consistent with other errors in this repo

```suggestion
    core.error('Must supply one of paths, globs');
    throw new Error();
```